src_new/adapters/prebid.js

```diff
@@ -557,16 +557,17 @@ function pushAdapterParamsInAdunits(adapterID, generatedKey, impressionID, keyCo
 		slotParams[key] = value;
 	});
 
-	//Setting Viewablity from localCache
-	var getAndParseFromLocalStorage = function getAndParseFromLocalStorage(key) {
+	//Setting Viewablity from localCache, if not found log error with undefined Bid Viewability Score
+	var getAndParseBVSFromLocalStorage = function getAndParseBVSFromLocalStorage(key) {
 		try{
 		return JSON.parse(window.localStorage.getItem(key));
```

Comment:
Add a check for getItem before accessing it. Otherwise everytime it will throw exception when getItem returns null or undefined.